internal/processor/otel/traces.go

```diff
@@ -566,6 +568,8 @@ func TranslateSpan(spanKind ptrace.SpanKind, attributes pcommon.Map, event *mode
 
 			switch kDots {
 			// http.*
+			case "rpc.grpc.status_code":
```

Comment:
```suggestion
			case semconv.AttributeRPCGRPCStatusCode:
```

Also can you please move this down to the "// rpc.*" section below?

Reply:
moved in 2cc84eb8f8ac7d6f80585604ef2453d333a26d53